client/plots/DEanalysis.js

```diff
@@ -536,6 +549,23 @@ add:
 				config
 			})
 		}
+
+		if (self.settings.gsea && self.app.opts.genome.termdbs) {
+			const gsea_params = {
```

Comment:
please add a comment in the code here:

backend assumes it will use msigdb, and it's not a parameter. later if genome.termdbs{} will contain multiple geneset dbs, on the ui will need to generate a <select> to choose one key of `termdbs{}`, and pass it as parameter here